### PR TITLE
Update default admin group for self managed install

### PR DESCRIPTION
**Pull request by VaishnaviHire**
**NOTE:** PR is opened against `isv-and-4907` branch

## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] For commits that came from upstream, `[UPSTREAM]` has been prepended to the commit message.
- [ ] JIRA link(s): https://issues.redhat.com/browse/RHODS-5394
- [ ] The Jira story is acked.
- [ ] Live build image: 
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work.
- [ ] QE contact acknowledges that this has been tested and is approved for merge.

### CODE REVIEW

**Comment by VaishnaviHire** [comment]
/hold

Until https://github.com/red-hat-data-services/odh-deployer/pull/260 PR is merged

**Review by andrewballantyne — APPROVED** [review]
Looks good to me from the Dashboard point of view. cc @lucferbux knows the code the best

File: deploy.sh [hunk 1]
```diff
@@ -361,6 +365,10 @@ fi
 kind="configmap"
 resource="odh-enabled-applications-config"
 object="odh-enabled-applications-config"
+
+
+sed -i "s|<admin_groups>|$ADMIN_GROUPS|g" odh-dashboard/configs/odh-dashboard-config.yaml
+
```
Comment (bdattoma) [inline deploy.sh:371]: same code is executed at line 390. Is that expected?

Comment (VaishnaviHire) [reply deploy.sh]: Thanks, removed the code

File: deploy.sh [hunk 2]
```diff
@@ -121,8 +121,12 @@ oc label namespace $ODH_MONITORING_PROJECT  $NAMESPACE_LABEL --overwrite=true ||
 # If rhodsquickstart CRD is found, delete it. Note: Remove this code in 1.19
 oc delete crd rhodsquickstarts.console.openshift.io 2>/dev/null || echo "INFO: Unable to delete Rhodsquickstart CRD"
 
+# Set rhods_self_managed to 0, if catalogsource not found.
+rhods_self_managed=1
+oc get catalogsource -n openshift-marketplace self-managed-rhods &> /dev/null || rhods_self_managed=0
```
[the remark this answers is not in the document]
Comment (VaishnaviHire) [reply deploy.sh]: Updated the variable

**Comment by VaishnaviHire** [comment]
> @VaishnaviHire Two small changes after your update

Thanks, updated the PR

**Pull request merged**